src/components/generics/inputs/InputOrientation.vue

```diff
@@ -1,5 +1,14 @@
 <template>
-  <div class="control">
+  <div
+    class="control"
+    :title="((value_.indexOf('E')>-1)?'E ':'')
```

Comment:
`value_` is an array, and method join will work like a charm : 

`:title="value_.join(', ')"`

And joining with comma+space will display orientation like this : 

`N, NE, E`

Reply:
Yes, it's a prettiest method...